src/routines.spec.ts

```diff
@@ -18,55 +18,67 @@ beforeAll(() => {
   jest.useFakeTimers();
 });
 
-const actions = {
-  a: mockAction('alpha', undefined, { letter: 'A' }),
-  b: mockAction('bravo', undefined, { letter: 'B' }),
-  c: mockAction('charlie', undefined, { letter: 'C' }),
-  e: mockAction('error'),
+const marbleMap: Record<string, any> = {
+  a: mockAction('[Mock] alpha', undefined, { letter: 'A' }),
+  b: mockAction('[Mock] bravo', undefined, { letter: 'B' }),
+  c: mockAction('[Mock] charlie', undefined, { letter: 'C' }),
+  e: mockAction('[Mock] error'),
   f: mockAction('[Mock] action', undefined, { letter: 'F' }),
 };
-const letters = {
-  A: 'A',
-  B: 'B',
-  C: 'C',
-  F: 'F',
-};
-const lengths = {
-  ...letters,
-  '5': 5,
-  '7': 7,
-};
+
+// Add letter representations to marble map
+for (let i = 'A'.charCodeAt(0); i < 'Z'.charCodeAt(0); i++) {
+  const letter = String.fromCharCode(i);
+  marbleMap[letter] = letter;
+}
+// Add number representations to marble map
+for (let i = 0; i < 10; i++) {
+  // Create marble representations of the 10 first characters
+  // eg. { A: 'A', B: 'B', ...etc }
+  const charCodeA = 'A'.charCodeAt(0);
+  const letter = String.fromCharCode(charCodeA + i);
+  marbleMap[letter] = letter;
+  // Create marble representations of the 10 first numbers from 0
+  marbleMap[`${i}`] = i;
+}
```

Comment:
I had to change these tests because it was using the lengths of the action types.. which obviously changes when you enforce the `ActionName` type.
Now the tests are more robust.